src/app/layout.tsx

```diff
@@ -17,20 +17,22 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import './globals.css';
-import { Inter } from 'next/font/google';
+/**
+ * React.Context, used by ThemeProvider, doesn't work server side so we're defaulting to client side rendering
+ */
+'use client';
```

Comment:
🤔interesting! this hasn't been necessary in daco nor platform, afaik.
is this a new requirement because of latest Emotion or NextJS?

Reply:
latest NextJS requirement.
Next 13 defaults to everything being treated as a server side component.

Reply:
React.context just doesn't work in the SSR way of doing things. You're meant to use it for SSR.
It's not a bug or a workaround or anything like that.